docs/develop/dotnet/data-encryption.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: data-encryption
+title: Date Encryption
+sidebar_label: Date encryption
+description: Date encryption
+keywords:
+  - sdk
+  - dotnet
+  - data encryption
+  - codec server
+  - converter
+tags:
+  - sdk
+  - dotnet
+  - data encryption
+  - codec server
+  - converter
+---
+
+This page shows the following:
+
+- How to use a custom Payload Codec
+- How to use a custom Payload Converter
+
+## Custom Payload Codec {#custom-payload-codec}
+
+**How to use a custom Payload Codec with the .NET SDK.**
+
+<!-- CONTENT-->
+
+## Payload conversion
+
+<!-- CONTENT-->
+
+### Conversion sequence {#conversion-sequence}
+
+<!-- CONTENT-->
+
+### Custom Payload Converter {#custom-payload-converter}
+
+**How to use a custom Payload Converter with the .NET SDK.**
+
+<!-- CONTENT-->
+
+**List of default Data Converter supports converting multiple types**
+
+<!-- CONTENT-->
+
+### Custom Type Data Conversion
+
+<!-- CONTENT-->
```

Comment:
Only the payload codec is related to encryption, none of the other sections are related to encryption in any way and are used by users for type conversion customizaion, not encryption.